File: Practice/16/Python/ex16.py
print("Введите колличество билетов Суслика: ")
n = int(input())
symbols = ""
print("Введите номера билетов каждый в новой строке: ")
i = 0
while i < n:
    tickets = input()
    if (tickets[0] == 'a') and (tickets[4] == '5') and (tickets[5] == '5') and (tickets[6] == '6') and (tickets[7] == '6') and (tickets[8] == '1'):
        symbols += tickets + " "
    i += 1
if symbols == "": print(-1)
else: print("Билеты Суслика:\n" + symbols)

File: Practice/16/C++/ex16/ex16/ex16.cpp
#include <iostream>
#include <string>

int main() {
	setlocale(LC_ALL, "ru");
	int n; std::cout << "Введите колличество билетов Суслика: " << std::endl;
	std::cin >> n;
	std::string tickets;
	std::string symbols = "";
	std::cout << "Введите номера билетов через пробел: " << std::endl;
	for (int i = 0; i < n; i++) {
		std::cin >> tickets;
		if ((tickets[0] == 'a') && (tickets[4] == '5') && (tickets[5] == '5') && (tickets[6] == '6') && (tickets[7] == '6') && (tickets[8] == '1')) {
			symbols = symbols + tickets + ' ';
		}
	}
	if (symbols == "") std::cout << -1;
	else std::cout << "Билеты Суслика:" << std::endl << symbols;
}